test/integration/consul-container/libs/utils/version.go

```diff
@@ -0,0 +1,43 @@
+package utils
```

Comment:
we can add 
```
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0
```

Reply:
This PR is for the`1.14` version which does not have these headers